lib/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -196,6 +196,10 @@ ProxyRenderDelegate::ProxyRenderDelegate(const MObject& obj)
 
     const MFnDependencyNode fnDepNode(obj);
     _proxyShape = static_cast<MayaUsdProxyShapeBase*>(fnDepNode.userNode());
+
+    _isUfeSelectionEnabled = (_proxyShape != nullptr) ?
```

Comment:
@mattyjams What's the remaining value of this variable given that we have inline access to it on proxy shape?

Reply:
Good point. None really, I suppose. We can just rely on the compiler to inline away the function call.

Rebased and updated to remove this member variable in commit dab4b46f5900c4a3bffd7d64a1dcb91fc0b949f3.